inc/Engine/Admin/PerformanceMonitoring/Database/Queries/PerformanceMonitoring.php

```diff
@@ -177,4 +177,28 @@ public function prune_old_items( int $to_keep ) {
 			$this->delete_item( $id );
 		}
 	}
+
+	/**
+	 * Change blurred rows into unblurred.
+	 *
+	 * @return bool|int
+	 */
+	public function unblur_blurred_rows() {
```

Comment:
```suggestion
	public function unblur_rows() {
```
I think that's enough to have a clear naming of the method. You wouldn't unblur something that isn't blurred anyway